sdk/src/Core/Amazon.Runtime/Pipeline/Handlers/Marshaller.cs

```diff
@@ -77,28 +79,31 @@ public override IAsyncResult InvokeAsync(IAsyncExecutionContext executionContext
         /// request and response context.</param>
         protected static void PreInvoke(IExecutionContext executionContext)
         {
-            var requestContext = executionContext.RequestContext;
-            requestContext.Request = requestContext.Marshaller.Marshall(requestContext.OriginalRequest);
-            requestContext.Request.AuthenticationRegion = requestContext.ClientConfig.AuthenticationRegion;
+            using (MetricsUtilities.MeasureDuration(executionContext.RequestContext, TelemetryConstants.SerializationDurationMetricName))
+            {
+                var requestContext = executionContext.RequestContext;
+                requestContext.Request = requestContext.Marshaller.Marshall(requestContext.OriginalRequest);
+                requestContext.Request.AuthenticationRegion = requestContext.ClientConfig.AuthenticationRegion;
 
 #if NETSTANDARD
-            var method = requestContext.Request.HttpMethod.ToUpperInvariant();
+                var method = requestContext.Request.HttpMethod.ToUpperInvariant();
 #else
             var method = requestContext.Request.HttpMethod.ToUpper(CultureInfo.InvariantCulture);
```

Comment:
If you are going to correct indentation in the prior case it should be done here as well.